pxr/usdImaging/usdImaging/cameraAdapter.cpp

```diff
@@ -199,6 +199,24 @@ UsdImagingCameraAdapter::TrackVariability(UsdPrim const& prim,
                    false)) {
         return;
     }
+
+    if ((*timeVaryingBits & HdCamera::DirtyParams) != 0)
+        return;
+
+    // If any other camera attributes are time varying
+    // we will assume all camera params are time-varying.
+    const std::vector<UsdAttribute> &attrs = prim.GetAttributes();
+    for (UsdAttribute const& attr : attrs) {
+        // Don't double-count transform attrs.
+        if (UsdGeomXformable::IsTransformationAffectedByAttrNamed(
+                attr.GetBaseName())) {
+            continue;
+        }
```

Comment:
Actually, I didn't understand this. Are you suggesting replacing the code with the clipping planes check? Adding the clipping planes check first? Why? :)

Reply:
Just like you "continue" if you run into an xform attribute, since you don't want it to affect "DirtyParams", similarly you don't want to process "clippingPlanes", since that is handled above as DirtyClipPlanes and shouldn't set "DirtyParams".

Reply:
Ah, gotcha